File: Python/2807.py
n = int(input())

anterior = 0
atual = 1
proximo = 0

sequencia = []

for i in range(n):
    sequencia.append(atual)
    proximo = anterior + atual
    anterior = atual
    atual = proximo

for i in range(n-1, -1, -1):
    if i == 0:
        print(f"{sequencia[i]}")
    else:
        print(f"{sequencia[i]}", end = " ")

File: C/2807.c
#include <stdio.h>

int main(){

    int n, sequencia[40], anterior = 0, atual = 1, proximo = 0;

    scanf("%i", &n);

    for(int i = 0; i<n; i++){
        sequencia[i] = atual;
        proximo = atual + anterior;
        anterior = atual;
        atual = proximo;
    }

    for(int i = n-1; i > -1; i--){
        if(i == 0){
            printf("%i\n", sequencia[i]);
        }else{
            printf("%i ", sequencia[i]);
        }
    }

    return 0;
}